R/github.R

```diff
@@ -60,15 +60,14 @@ get_repo_data <- function(repo){
 }
 
 extract_repo <- function(path){
-  if(grepl("^git@github.com", path)){
-    path <- sub("^git@github.com", "https://github.com", path)
-  } else if(grepl("^http://github.com", path)){
-    path <- sub("^http://github.com", "https://github.com", path)
-  }
-  if(!all(grepl("^https://github.com", path))){
+  github_url <- "https://github.com/"
+  path <- sub("^git@github[.]com:", github_url, path)
+  path <- sub("^http://github[.]com/", github_url, path)
+  path <- sub("^git://github[.]com/", github_url, path)
+  if(!all(grepl("^https://github.com.*[.]git$", path))){
```

Comment:
Note that remotes don't have to end in `.git`. A remote can also simply be `https://github.com/user/repo`.